doc/quick-start-user-guide.md

```diff
@@ -71,9 +77,41 @@ etcdcurls = "http://192.168.56.101:2379"
 etcdpurls = "http://192.168.56.101:2380"
 ```
 
+**Authentication:** In glusterd2 REST API authentication is enabled by default. To disable rest authentication add `restauth=false` in Glusterd2 config file(Default path is
```

Comment:
The default path of config file should be a separate subsection that comes before `Authentication` section